src/docsfy/postprocess.py

```
@@ -15,12 +16,168 @@
+            # Escape markdown special chars in the title for safe link text
+            safe_title = (
+                canonical_title.replace("\\", "\\\\")
+                .replace("[", "\\[")
+                .replace("]", "\\]")
+            )
+            return f"{see_prefix}[{safe_title}]({target_slug}.html)"
+
+        # Split content into code and non-code segments, only linkify non-code
+        parts = _CODE_BLOCK_RE.split(content)
+        new_parts: list[str] = []
+        for i, part in enumerate(parts):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)